api/editor/api-editor/src/main/java/ca/bc/gov/tno/ApiEditor.java

```diff
@@ -1,17 +1,34 @@
+/**
+ * API Editor provides a RESTful web service for the TNO application.
+ */
 package ca.bc.gov.tno;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
+/**
+ * ApiEditor class, provides endpoints for the editor web application,
+ * and provides secure access to data sources and 3rd party services.
+ */
 @EnableJpaRepositories(basePackages = { "ca.bc.gov.tno.dal.db" })
 @EntityScan(basePackages = { "ca.bc.gov.tno.dal.db" })
-@SpringBootApplication(scanBasePackages = { "ca.bc.gov.tno", "ca.bc.gov.tno.keycloak", "ca.bc.gov.tno.dal.db",
-		"ca.bc.gov.tno.azure", "ca.bc.gov.tno.elastic", "ca.bc.gov.tno.kafka" })
-public class ApiEditor {
+@SpringBootApplication(scanBasePackages = {
```

Comment:
Attempting to resolve linting issues, but unable to to get the IDE to use spaces instead of tabs...